scripts/setup_env_linux_debian.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# setup dev env on debian
+
+# check if sudo permission is not available
+if [ "$EUID" -ne 0 ]; then
+    echo "sudo permission is not available"
+    exit
+fi
+
+# install system-wide dependencies 
+apt-get install python3 python-pip
```

Comment:
so when you are using pyenv, apt-get install python3 is not required. You can kill it. also, install curl and git as well.